### ISSUE TITLE
Open the keyboard when selecting the tool

### ISSUE THREAD

**Issue by Saeeed-B** [JiHong88/SunEditor#953]
@JiHong88 
Hello
On mobile, the keyboard opens automatically when we select one of the tools. And we have to close the keyboard again and work with the tools

For example, suppose we select a text and want to change its color, at which point the keypad opens.

**Comment by Saeeed-B** [comment]
@JiHong88 
Apparently when a tool is clicked, a position is given to the `cursor`.

And when given  `cursor` new position, the keyboard opens automatically.

This bug is easily visible, you can go to the following address with your mobile phone and select a text and select a tool.
You see the keyboard opens.
[http://suneditor.com/sample/html/examples.html](http://suneditor.com/sample/html/examples.html)

**Comment by JiHong88** [comment]
It is a feature of the browser that the keyboard automatically opens when the cursor is positioned at a new location.
This is not a bug.
If you don't want the keyboard to open, you don't have to give it focus, then you can't change the style of the text.
This is also true for other editors.

**Comment by Saeeed-B** [comment]
The same thing happens when we click on more buttons in responsive mode

**Comment by JiHong88** [comment]
Yes, the focus should be maintained even when the More button is pressed.
This is an unavoidable action.
